### DISCUSSION TITLE
How to get betas?

### DISCUSSION THREAD

**Discussion by Fluffy728** [authelia/authelia#8525] [Q&A] [answered]
How do we get pre releases?

**Comment by Fluffy728** [comment]
Oh just relised duplicate of https://github.com/authelia/authelia/discussions/6204

**Comment by Fluffy728** [comment]
Fine…

**Comment by james-d-elliott** [comment, marked as answer]
Each PR has a listed docker build in the [PR](https://github.com/authelia/authelia/pulls), and the buildkite steps list artifacts associated with the build event. The `master` branch is also published as the `master` tag. Please also take note that if you upgrade to one of these that it's generally unsupported and the instructions to recover a database after corruption are to "restore a backup or delete it and start from the beginning". Support requests for databases corrupted in this way will likely be summarily refused.

It should also be noted we do not have any current pre-release builds but you're welcome to try any particular [PR](https://github.com/authelia/authelia/pulls) or master out at your own risk and following the applicable instructions in the linked topic.

**Comment by Fluffy728** [comment]
Thanks for telling me that!
